callstats-java-sdk/src/main/java/io/callstats/sdk/BridgeStatusInfo.java

```diff
@@ -0,0 +1,75 @@
+package io.callstats.sdk;
+
+public class BridgeStatusInfo {
+	private float cpuUsage;
+	private float memoryUsage;
+	private long sentBytes;
+	private long receivedBytes;
+	private float intervalLoss;
+	private float totalLoss;
+	private float avgIntervalRtt;
+	private float avgIntervalJitter;
+	
+	public BridgeStatusInfo(float cpuUsage, float memoryUsage, long sentBytes,
```

Comment:
If we already have a "BridgeStatusInfoBuilder" to help building a "BridgeStatusInfo" object, I wonder if it is OK to set the access modifier for this constructor to be 'protected' to discourage people from directly creating  BridgeStatusInfo instance with this long parameter-list constructor. What is the general practice when a class has a builder?